schema.graphql

```diff
@@ -214,6 +217,11 @@ type FixedProductMarketMaker @entity {
 
   "Addresses which are supplying liquidity to the market maker"
   poolMembers: [FpmmPoolMembership!] @derivedFrom(field: "pool")
+
+  "Total locked value in this market"
```

Comment:
@cristiangpr let's use the lingo used to describe collateralVolume because it adds "...in USDC base units" and also add scaledOpenInterest with the description having "...scaled by 10^6"